libr/core/anal_tp.c

```diff
@@ -586,16 +688,15 @@ R_API void r_core_anal_type_match(RCore *core, RAnalFunction *fcn) {
 			if (i < bblist_size) {
 				bb = r_anal_get_block_at (core->anal, bb_addr);
 				if (!bb) {
-					R_LOG_WARN ("basic block at 0x%08"PFMT64x" was removed during analysis", bblist[i]);
+					R_LOG_WARN ("basic block at 0x%08"PFMT64x" was removed during analysis", *RVecUT64_at (&bblist, i));
 					retries--;
-					free (bblist);
 					goto repeat;
 				}
 			}
 
 			bool userfnc = false;
-			Sdb *trace = anal->esil->trace->db;
-			cur_idx = sdb_num_get (trace, "idx", 0);
+			// Sdb *trace = anal->esil->trace->db;
+			cur_idx = etrace_index (etrace); // sdb_num_get (trace, "idx", 0);
```

Comment:
rm comments? (also in other places)

Reply:
i kept the comments for debugging purposes. better wait until its green to rm